From 9515448761739d6186e7d07da5b47e368753528c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 1 Sep 2020 11:34:33 -0700
Subject: [PATCH] make: Pass LDFLAGS when building shared objects

OE passes flags like hash-style via LDFLAGS which alters the linker
defaults, its important to have LDFLAGS in link step even if compiler
driver is used to do linking

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 mk/Rules.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/Rules.mk b/mk/Rules.mk
index b480bd6..db65125 100644
--- a/mk/Rules.mk
+++ b/mk/Rules.mk
@@ -344,7 +344,7 @@ $(BEARSSLLIB): $(OBJDIR) $(OBJ)
 	$(AR) $(ARFLAGS) $(AROUT)$(BEARSSLLIB) $(OBJ)
 
 $(BEARSSLDLL): $(OBJDIR) $(OBJ)
-	$(LDDLL) $(LDDLLFLAGS) $(LDDLLOUT)$(BEARSSLDLL) $(OBJ)
+	$(LDDLL) $(LDDLLFLAGS) $(LDFLAGS) $(LDDLLOUT)$(BEARSSLDLL) $(OBJ)
 
 $(BRSSL): $(BEARSSLLIB) $(OBJBRSSL)
 	$(LD) $(LDFLAGS) $(LDOUT)$(BRSSL) $(OBJBRSSL) $(BEARSSLLIB)
-- 
2.28.0

